gen-ai-rag-vertex-ai-vector-search/ingestion.tf

```diff
@@ -0,0 +1,77 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Aspects to setup ingestion from Cloud Storage bucket to processing job.
```

Comment:
Don't we also need a "google_storage_notification" resource to enable notifications to be sent to the Pub/Sub topic when a new object is added or updated in the GCS bucket ?

https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_notification#event_types-1

Reply:
I thought I would be handling this through ingestion_data_source_settings, but I'd have to check what the differences are here.

Reply:
I've established that using a base of https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/functions/pubsub/main.tf, adding https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_notification#example-usage to put data into the topic, and adapting the JavaScript sample to Python (on request), I have a working ingestion pipeline